File: leetcode/remove-trailing-zeros-from-a-string.cpp
#include <bits/stdc++.h>
using namespace std;


class Solution {
public:
    string removeTrailingZeros(string num) {
        int lnznindex;
        string fo;
        for(int x=num.size()-1;x>-1;x--){
            if(num[x]!='0'){
                lnznindex = x;
                break;
            }
        }
        if (lnznindex == num.size()-1){
            return num;
        }
        for(int x=0;x<=lnznindex;x++){
            fo += num[x];
        }
        return fo;
        
        
    }
};

File: leetcode/remove-trailing-zeros-from-a-string.py
class Solution:
    def removeTrailingZeros(self, num: str) -> str:
        fo = ""
        for x in range(len(num)):
            if num[x] != '0':
                ind = x
        if ind == len(num)-1:
            return num
        for x in range(0, ind+1):
            fo += num[x]
        return fo